client/test/components/notebook/convert.test.ts

```diff
@@ -0,0 +1,250 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  generateCodeListFromPythonNotebook,
+  generateCodeListFromSASNotebook,
+  generateFlowDataNode,
+  generateFlowDataSwimlane,
+} from "../../../src/components/ContentNavigator/convert";
```

Comment:
In general, we shouldn't be exporting functions from files _just_ for the purposes of testing. In this case, I'm wondering if we can test all the things through `convertNotebookToFlow` since that is the only function that is used in non-test code.

Reply:
done in 5f3a455e8798a4a89997b40e894f660e63b00135